Wrapping/Generators/Python/itk/support/helpers.py

```diff
@@ -178,6 +178,73 @@ def image_filter_wrapper(*args, **kwargs):
     return image_filter_wrapper
 
 
+def python_to_js(name):
+    import itk
+
+    def _long_type():
+        if os.name == "nt":
+            return "int32"
+        else:
+            return "int64"
+
+    _python_to_js = {
+        itk.SC: "int8",
+        itk.UC: "uint8",
+        itk.SS: "int16",
+        itk.US: "uint16",
+        itk.SI: "int32",
+        itk.UI: "uint32",
+        itk.F: "float32",
+        itk.D: "float64",
+        itk.B: "uint8",
+        itk.SL: _long_type(),
+        itk.UL: "u" + _long_type(),
+        itk.SLL: "int64",
+        itk.ULL: "uint64",
+    }
+
+    return _python_to_js[name]
+
+
+def js_to_python(name):
+    def _long_type():
+        if os.name == "nt":
+            return "LL"
+        else:
+            return "L"
+
+    _js_to_python = {
+        "int8": "SC",
+        "uint8": "UC",
+        "int16": "SS",
+        "uint16": "US",
+        "int32": "SI",
+        "uint32": "UI",
+        "int64": "S" + _long_type(),
+        "uint64": "U" + _long_type(),
+        "float32": "F",
+        "float64": "D",
+    }
+
+    return _js_to_python[name]
+
+
+def pixelType_to_prefix(name):
+    _pixelType_to_prefix = {
+        "Scalar": "",
+        "RGB": "RGB",
+        "RGBA": "RGBA",
+        "Offset": "O",
+        "Vector": "V",
+        "CovariantVector": "CV",
+        "SymmetricSecondRankTensor": "SSRT",
+        "FixedArray": "FA",
+        "Array": "A",
```

Comment:
Should `VLV` go here? VariableLengthVector might need to be supported here, maybe elsewhere too.